bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -192,6 +192,7 @@ class ReplicaConfig : public concord::serialize::SerializableFactory<ReplicaConf
     serialize(outStream, batchingFactorCoefficient);
 
     serialize(outStream, publicKeysOfReplicas);
+    // serialize(outStream, publicKeysOfClients);
```

Comment:
Why publicKeysOfClients can't be serialized/deserialized?

Reply:
Actually, This one is removed on my local branch but didn't push yet. I want to perform it in another task.
When uncommenting it just doesn't work.
The current serilize/deserialie functions do not handle a type:
std::set<std::pair<const std::string, std::set<uint16_t>>>

So I want to implement this in another task, since It will take at least a day or more to do, and from my conversations with people, it isn't a high priority task for now.